docs/docs-content/clusters/edge/edge-host-management/access-console.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_label: "Access Edge Management Console"
+title: "Access Edge Management Console"
+description: "Instructions for accessing the Edge Management Console."
+hide_table_of_contents: false
+sidebar_position: 32
+tags: ["edge"]
+---
+
+The Edge Management Console (EMC) allows you to manage Edge hosts in your network locally, upload content such as images
+and packs, and create Edge clusters without connections to a centralized Palette installation.
+
+:::preview
+
+:::
+
+This page guides you through how access the console, and log in, and manage your username and password.
+
+## Log in to Edge Management Console
+
+### Prerequisites
+
+- An Edge host with installed with Edge Installer 4.3 or later using the `airgapped` install mode.
+
+- You have completed or skipped the initial configuration of the Edge host. For more information, refer to
+  [Edge Host Initial Configuration](../site-deployment/initial-setup.md).
+
+- Network access to the Edge host's port where EMC is exposed. The default port is 5080.
+
+### Instructions
+
+1. Ensure the Edge host is powered on.
+
+2. In your browser, go to `https://HOST_IP:5080`. Replace `HOST_IP` with the IP address of your Edge host. If you have
+   access to the Edge host terminal, the address of the EMC console is displayed on the terminal screen. If you have
+   changed the default port of the console, replace `5080` with the EMC port.
+
+3. You'll be prompted to log in. Enter your username and password to log in.
+
+### Validate
+
+1. A successful login directs you to the Edge management console. You will see an overview of your Edge host on your
```

Comment:
```suggestion
1. A successful login directs you to the Edge Management Console. You will see an overview of your Edge host on your
```
Should it be "Edge Management Console" because that's how instances are written?